src/content/docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems.mdx

```diff
@@ -48,9 +48,9 @@ The <DoNotTranslate>**Transactions**</DoNotTranslate> page also may include link
 
 ## Wall-clock time [#wall_clock_time]
 
-Wall-clock time is the amount of time recorded by the clock. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
+Wall-clock time measures "real-time elapsed" during a specific transaction. For example, let's say you're an engineer responsible for managing the checkout experience on an e-commerce site. You would like to understand how long it takes for a customer to add an item to their cart. The transaction took 15 seconds to complete but one minute in the real world or on the "wall clock" for the customer. This discrepancy in time could be because the function had to wait for network calls or other inputs and outputs, and that waiting time isn't considered in the system's transaction time. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
```

Comment:
```suggestion
Wall-clock time measures "real-time elapsed" during a specific transaction. For example, let's say you're an engineer responsible for managing the checkout experience on an ecommerce site. You'd like to understand how long it takes for a customer to add an item to their cart. The transaction took 15 seconds to complete but one minute in the real world or on the "wall clock" for the customer. This discrepancy in time could be because the function had to wait for network calls or other inputs and outputs, and that waiting time isn't considered in the system's transaction time. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
```

Reply:
Great example and clarified things for me. I wonder if we should break up this chunk of text though, maybe introducing the time content with bullet points so it looks like:

> ...an item to their cart.
>
>  * The transaction took 15 seconds to complete, but one minute passed from the customer's point of view.
> * The time elapsed from the customer's point of view is called the "wall clock" for the customer.
>
> This discrepancy.... yadda yadda yadda